.github/workflows/docker-hub-parachain.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           images: |
             zeitgeistpm/zeitgeist-node-parachain
           tags: |
-            type=sha
+            type=semver,pattern={{version}}
```

Comment:
> This line of code is saying that the tags for the Docker images being built are going to be in the format of a semantic version (i.e. 1.0.0, 2.3.1, etc.) and the specific version number will be determined by the {{version}} variable.

*ChatGPT

Instead of a `sha` hash, we use a semantic version for the docker image.

Reply:
where does this version variable set?

Reply:
@vivekvpandya The GitHub Action [metadata-action](https://github.com/docker/metadata-action#typesemver) does set it.